app/classifier/tutorial.jsx

```diff
@@ -263,10 +272,17 @@ export default class Tutorial extends React.Component {
         lang={this.props.locale}
         style={tutorialStyle}
       >
-        <MediaCard className="tutorial-step" src={mediaCardSrc}>
-          <Markdown>{this.props.translation.steps[this.state.stepIndex].content}</Markdown>
-          <hr />          
-        </MediaCard>
+        <div
+          aria-live="polite"
+          className="tutorial-step"
+        >
+          <MediaCard
+            src={mediaCardSrc}
+          >
+            <Markdown>{this.props.translation.steps[this.state.stepIndex].content}</Markdown>
+            <hr />
+          </MediaCard>
+        </div>
         <p style={{ textAlign: 'center' }}>
           {(this.state.stepIndex === this.props.tutorial.steps.length - 1) ?
             <button type="submit" className="major-button">
```

Comment:
I can press the "Let's go!" button on VoiceOver (i.e., control + option + space), but the action does not close the tutorial.

Reply:
It turns out `Dialog.alert()` only works if the modal content is a form. It hijacks the form's `submit` event to close the tutorial. Now that I've replaced `<form>` with `<section>`, that doesn't work.

The solution is to remove `Dialog.alert()` completely.